portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,11 +47,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
-    // Get the input from the form.
+    // Get the input from the form and log time of comment
     String commentText = request.getParameter("comment-text");
+    long timestamp = System.currentTimeMillis();
 
     commentsList.add(commentText);
 
+    // Create entity with comment text and timestamp info
+    Entity taskEntity = new Entity("Comment");
+    taskEntity.setProperty("text", commentText);
+    taskEntity.setProperty("timestamp", timestamp);
```

Comment:
nit: All the strings used here ("comment-text", "Comment", "text", "timestamp") should be made `private static final String` as a class level variable because they will have broader implications (eg. they will be used latter to get back the data from the Datastore).